### PR TITLE
#11990: Replace ttnn.experimental.tensor.* to ttnn.* in ttnn folder

### PR DESCRIPTION
**Pull request by bharane-ab**
### Ticket
Link to Github Issue #11990
Master issue #11646 

### Problem description
update ttnn.experimental with ttnn ops

### What's changed
python files have been updated with TTNN support

### Checklist
- [x] [post commit ci](https://github.com/tenstorrent/tt-metal/actions/runs/10627098883)

### CODE REVIEW

**Review by Aswinmcw — APPROVED** [review]

**Review by ayerofieiev-tt — APPROVED** [review]

File: tests/ttnn/ttnn_utility_fuction.py [hunk 1]
```diff
@@ -6,44 +6,38 @@
...
+                grid_coord_1 = ttnn.CoreCoord(core_grid_1.x - 1, core_grid_1.y - 1)
+                grid_coord_2 = ttnn.CoreCoord(core_grid_2.x - 1, core_grid_2.y - 1)
+                return ttnn.CoreRangeSet(
                     {
-                        ttnn.experimental.tensor.CoreRange(ttnn.experimental.tensor.CoreCoord(0, 0), grid_coord_1),
-                        ttnn.experimental.tensor.CoreRange(
-                            ttnn.experimental.tensor.CoreCoord(0, grid_coord_2.y), grid_coord_2
-                        ),
+                        ttnn.CoreRange(ttnn.CoreCoord(0, 0), grid_coord_1),
+                        ttnn.CoreRange(ttnnCoreCoord(0, grid_coord_2.y), grid_coord_2),
```
Comment (mcw-anasuya) [inline tests/ttnn/ttnn_utility_fuction.py:29]: ttnn.CoreCoord

File: ttnn/ttnn/operations/conv/untilize_with_halo_config_generation_and_validation.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 import numpy as np
 from loguru import logger
 
-from tt_lib._internal.comparison_funcs import comp_equal
+from black._internal.comparison_funcs import comp_equal
```
Comment (umadevimcw) [inline ttnn/ttnn/operations/conv/untilize_with_halo_config_generation_and_validation.py:9]: is it black?

File: ttnn/tt_lib/fused_ops/conv.py [hunk 3]
```diff
@@ -369,7 +362,7 @@ def resnet50_first_conv(
     )
 
     def conv_(activation):
-        # assert(activation.get_layout() == tensor.Layout.ROW_MAJOR)
+        # assert(activation.get_layout() == ttnn.ROW_MAJOR_LAYOUT)
```
Comment (umadevimcw) [inline ttnn/tt_lib/fused_ops/conv.py:365]: Remove commented code if its not needed

**Review by umadevimcw — CHANGES_REQUESTED** [review]

File: ttnn/tt_lib/fused_ops/conv.py [hunk 4]
```diff
@@ -3,7 +3,6 @@
 # SPDX-License-Identifier: Apache-2.0
 
 from typing import List, Union
-from tt_lib import tensor, operations
```
Comment (mcw-anasuya) [inline ttnn/tt_lib/fused_ops/conv.py:6]: Keep `from tt_lib import tensor` since there are tensor.convert_conv* functions.
These functions will be handled in a different PR.

File: ttnn/ttnn/operations/pool.py [hunk 5]
```diff
@@ -26,7 +26,6 @@
 from typing import Union, Tuple, Dict
 
 from tt_lib.utils import _nearest_32
-import tt_lib as ttl
```
Comment (mcw-anasuya) [inline ttnn/ttnn/operations/pool.py:29]: imported ttnn twice

File: ttnn/tt_lib/fallback_ops/conversion_wrapper.py [hunk 6]
```diff
@@ -2,7 +2,6 @@
 
 # SPDX-License-Identifier: Apache-2.0
 
-from tt_lib import tensor as ttl_tensor
 import ttnn
```
[the remark this answers is not in the document]
Comment (bharane-ab) [reply ttnn/tt_lib/fallback_ops/conversion_wrapper.py:5]: It's previously imported by someone in file. I will clean it up

File: ttnn/tt_lib/fused_ops/conv.py (hunk 4 above)
Comment (bharane-ab) [reply ttnn/tt_lib/fused_ops/conv.py:6]: ok then I will undo those changes

**Review by mcw-anasuya — APPROVED** [review]

**Review by umadevimcw — APPROVED** [review]

**Review by eyonland — APPROVED** [review]
